src/main/java/org/team1540/base/power/PowerManager.java

```diff
@@ -64,43 +69,49 @@ public void run() {
     }
   }
 
-  // Separate method to block command registration/unregistration while actually scaling the power.
+  /**
+   * Separate method to block PowerManageable registration/unregistration while actually scaling the
```

Comment:
This won't block it: synchronized methods simply prevent multiple instances of _that method_ from running. May as well just map those things in a synchronized collection.